mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -158,6 +183,16 @@ class GeneralizedLinearRegression @Since("2.0.0") (@Since("2.0.0") override val
   setDefault(family -> Gaussian.name)
 
   /**
+    * Sets the value of param [[variancePower]].
+    * Used only when family is "tweedie".
+    *
+    * @group setParam
+    */
+  @Since("2.2.0")
+  def setVariancePower(value: Double): this.type = set(variancePower, value)
+  setDefault(variancePower -> 1.5)
```

Comment:
Done. change default variancePower to 0.0, which will use Gaussian (with default identity link)